lnrpc/rpc.proto

```diff
@@ -2120,14 +2121,40 @@ message WalletBalanceResponse {
     int64 unconfirmed_balance = 3;
 }
 
+message Amount {
+    // Value denominated in satoshis.
+    int64 sat = 1;
```

Comment:
@yyforyongyu as you pointed out, i think it makes sense to use `uint64` here and below

Reply:
yep, now switched.